src/averageAges.js

```diff
@@ -37,9 +45,23 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
```

Comment:
Similar to the previous function, consider renaming this function to accurately describe its purpose.